[2024-06-24 14:26:09,454] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2022-11-26T00:00:00+00:00 [queued]>
[2024-06-24 14:26:09,514] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2022-11-26T00:00:00+00:00 [queued]>
[2024-06-24 14:26:09,518] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:26:09,519] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-24 14:26:09,520] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:26:09,597] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-11-26T00:00:00+00:00
[2024-06-24 14:26:09,607] {standard_task_runner.py:52} INFO - Started process 12394 to run task
[2024-06-24 14:26:09,613] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2022-11-26T00:00:00+00:00', '--job-id', '6086', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp6uk5kyk2', '--error-file', '/tmp/tmpl2qrcomk']
[2024-06-24 14:26:09,629] {standard_task_runner.py:77} INFO - Job 6086: Subtask second_task
[2024-06-24 14:26:09,805] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2022-11-26T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-06-24 14:26:09,970] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-26T00:00:00+00:00
[2024-06-24 14:26:09,976] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-06-24 14:26:09,978] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2024-06-24 14:26:10,001] {bash.py:169} INFO - Output:
[2024-06-24 14:26:10,015] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2024-06-24 14:26:10,017] {bash.py:177} INFO - Command exited with return code 0
[2024-06-24 14:26:10,311] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20221126T000000, start_date=20240624T142609, end_date=20240624T142610
[2024-06-24 14:26:11,201] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24 14:26:11,258] {local_task_job.py:146} INFO - Task exited with return code 0
